encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -73,8 +73,8 @@ func New(impl Interface, opts ...<$thrift>.RegisterOption) []<$transport>.Regist
 	h := handler{impl}
 	service := <$thrift>.Service{
 		Name: "<.Service.Name>",
-		Methods: map[string]<$thrift>.Handler{
-			<range .Service.Functions>"<.ThriftName>": <$thrift>.HandlerFunc(h.<.Name>),
+		Methods: map[string]<$thrift>.UnaryHandler{
+			<range .Service.Functions><if not .OneWay>"<.ThriftName>": <$thrift>.UnaryHandlerFunc(h.<.Name>),<end>
```

Comment:
Technically an unrelated change but it should be fine. Make sure that the
glide.yaml depends on thriftrw 0.4 otherwise this won't work.